File: Professional Image Modifier 2/[VB]-Professional Image Modifier 2/VB/ImageFunctions/Forms/FrmProcessing.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms

Namespace ImageFunctions.Forms
	Partial Public Class FrmProcessing
		Inherits Form

		Public Property description() As String

		Public Sub New(ByVal description As String)
			InitializeComponent()
			lblDescription.Text = description

		End Sub


	End Class
End Namespace

File: Professional Image Modifier 2/[C#]-Professional Image Modifier 2/C#/ImageFunctions/Forms/FrmProcessing.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace ImageFunctions.Forms
{
	public partial class FrmProcessing : Form
	{
		public string description { get; set; }

		public FrmProcessing(string description)
		{
			InitializeComponent();
			lblDescription.Text = description;

		}


	}
}
